src/components/Footer/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { genericPropTypes, genericPropsDefaults } from '../../utils/prop-types'
+import { FooterContainer, GreyPanelRow, MASBannerRow, ContactPanelColumn, ContactPanelRow,
+            FooterLinks,
+            SocialMediaCounter,
+            Images,
+            LanguageText} from './StyledFooter'
+import { Col } from '../Grid'
+
+
+const Footer = ({as, direction, ...rest}) => {
+    return(
+        
+        <FooterContainer fluid as="footer" {...rest}>
+            <ContactPanelRow debug noGutter>
+                <ContactPanelColumn sizes={{xs: 12, md: 4}} debug>webchati</ContactPanelColumn>
+                <ContactPanelColumn sizes={{xs: 12, md: 4}}  debug>whatsapp</ContactPanelColumn>
+                <ContactPanelColumn  sizes={{xs: 12, md: 4}} debug>phone</ContactPanelColumn>
+            </ContactPanelRow>
+                
+        
+            <GreyPanelRow>
+               
+                <SocialMediaCounter  sizes={{xs: 12, sm: 6}} debug>twiter</SocialMediaCounter>
+               
+                <FooterLinks  sizes={{xs: 12, md: 4}} debug>links</FooterLinks>
+                <Images  sizes={{xs: 12, md: 2}} debug>images</Images>
```

Comment:
Like the SocialMediaCounter, this should use the sm breakpoint so all elements break at the same resolution